### PR TITLE
RPF-119: e2e tests

### PR DESCRIPTION
**Pull request by rzcoder**

### CODE REVIEW

File: packages/e2e-tests/tests/mint-bid-accepbid.test.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+					"@type": "XTZ",
+				}
+			}
+			throw new Error("Wrong blockchain")
+		},
+	},*/
+]
+
+describe("mint-bid-acceptBid", () => {
+	for (const suite of suites) {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/mint-bid-accepbid.test.ts:102]: use describe.each instead

File: packages/e2e-tests/tests/mint-sell-buy.test.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+					contract: toContractAddress("FLOW:A.7e60df042a9c0868.FlowToken"),
+				}
+			}
+			throw new Error("Wrong blockchain")
+		},
+	},*/
+]
+
+describe("mint-sell-buy", () => {
+	for (const suite of suites) {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/mint-sell-buy.test.ts:127]: use describe.each instead

File: packages/e2e-tests/tests/mint-sell-cancel.test.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+					contract: toContractAddress("FLOW:A.7e60df042a9c0868.FlowToken"),
+				}
+			}
+			throw new Error("Wrong blockchain")
+		},
+	},*/
+]
+
+describe("mint-sell-cancel", () => {
+	for (const suite of suites) {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/mint-sell-cancel.test.ts:97]: use describe.each instead

File: packages/e2e-tests/tests/deploy-min.test.ts [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+				lazyMint: false,
+				supply: 1,
+			}
+		},
+	},
+]
+
+
+describe("deploy-mint", () => {
+	for (const suite of suites) {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/deploy-min.test.ts:85]: use describe.each instead

File: packages/e2e-tests/tests/common/atoms-tests/deploy-collection.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import type { IRaribleSdk } from "@rarible/sdk/src/domain"
+import type { BlockchainWallet } from "@rarible/sdk-wallet"
+import type { DeployResponse, DeployTokenRequest } from "@rarible/sdk/src/types/nft/deploy/domain"
+
+/**
+ * Buying an nft
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/common/atoms-tests/deploy-collection.ts:6]: Why is it buying nft?

File: packages/e2e-tests/tests/deploy-min.test.ts [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+	blockchain: Blockchain,
+	wallet: BlockchainWallet,
+	deployRequest: (address: UnionAddress) => DeployTokenRequest
+	mintRequest: (address: UnionAddress) => MintRequest
+}[] = [
+	{
+		blockchain: Blockchain.ETHEREUM,
+		wallet: getEthereumWallet(),
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		deployRequest: (walletAddress: UnionAddress) => {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/deploy-min.test.ts:23]: is it necessary to use deployRequest as a function?

File: packages/e2e-tests/tests/common/wallet.ts [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+export function getEthereumWallet(pk?: string): EthereumWallet {
+	const { web3, wallet } = initProvider(pk)
+	const ethereum = new Web3Ethereum({
+		web3: web3,
+		from: wallet.getAddressString(),
+	})
+	return new EthereumWallet(ethereum)
+}
+
+export function getTezosWallet(walletNumber: number = 0): TezosWallet {
```
Comment (ex1st0r) [inline packages/e2e-tests/tests/common/wallet.ts:19]: This logic is not explicitly

File: packages/e2e-tests/.babelrc [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    "@babel/preset-env"
+  ],
+  "plugins": [
+    "@babel/transform-runtime",
+    "@babel/plugin-transform-typescript",
+    "@babel/proposal-class-properties",
+    "@babel/proposal-object-rest-spread",
+    "@babel/plugin-proposal-optional-chaining"
+  ]
+}
```
Comment (ex1st0r) [inline packages/e2e-tests/.babelrc:13]: Is it needed to use Babel in this project?

File: packages/e2e-tests/tests/deploy-min.test.ts (hunk 6 above)
Comment (rzcoder) [reply packages/e2e-tests/tests/deploy-min.test.ts:23]: Yes, until it can depend on the arguments

**Pull request merged**